packages/suite/src/components/suite/NavigationBar/components/NavigationActions/index.tsx

```diff
@@ -128,6 +130,20 @@ const NavigationActions = (props: Props) => {
                 </>
             )}
 
+            {allowPrerelease &&
```

Comment:
This is not required because the desktop app has a width limit. User cannot see the menu. (same applies for tor)

Reply:
I do not understand. But lets leave it as it is